storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/queries/Query.java

```diff
@@ -477,6 +477,50 @@ public CompleteBuilder limit(@Nullable String limit) {
         }
 
         /**
+         * Optional: Specifies {@code LIMIT} clause.
+         * <p>
+         * Optional specifier that limits the number of rows returned by the query.
+         *
+         * @param limit positive number of rows returned by the query.
+         * @return builder.
+         * @see Query#limit()
```

Comment:
And link to the SQLite documentation https://www.sqlite.org/lang_select.html#limitoffset